bgpd/bgp_pbr.h

```diff
@@ -0,0 +1,268 @@
+/*
+ * BGP pbr
+ * Copyright (C) 6WIND
+ *
+ * This file is part of FRR.
+ *
+ * FRR is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2, or (at your option) any
+ * later version.
+ *
+ * FRR is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; see the file COPYING; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+#ifndef __BGP_PBR_H__
+#define __BGP_PBR_H__
+
+#include "nexthop.h"
+#include "zclient.h"
+
+/* flowspec case: 0 to 3 actions maximum:
+ * 1 redirect
+ * 1 set dscp
+ * 1 set traffic rate
+ */
+#define ACTIONS_MAX_NUM 4
+typedef enum {
+	ACTION_TRAFFICRATE = 1,
+	ACTION_TRAFFIC_ACTION = 2,
+	ACTION_REDIRECT = 3,
+	ACTION_MARKING = 4,
+	ACTION_REDIRECT_IP = 5
+} action_t;
+
+#define TRAFFIC_ACTION_SAMPLE     (1 << 0)
+#define TRAFFIC_ACTION_TERMINATE  (1 << 1)
+#define TRAFFIC_ACTION_DISTRIBUTE (1 << 2)
+
+#define	OPERATOR_COMPARE_LESS_THAN    (1<<1)
```

Comment:
Please remove these tabs, they serve no purpose.

Reply:
removed the tabs between define and OPERATOR..